src/main/java/com/faforever/client/teammatchmaking/TeamMatchmakingController.java

```diff
@@ -43,25 +58,68 @@
   @FXML
   public StackPane teamMatchmakingRoot;
   @FXML
-  public JFXListView<PartyMember> playerListView;
-  @FXML
   public JFXButton leavePartyButton;
   @FXML
   public JFXButton readyButton;
   @FXML
   public Label refreshingLabel;
-
+  public ToggleButton uefButton;
+  public ToggleButton cybranButton;
+  public ToggleButton aeonButton;
+  public ToggleButton seraphimButton;
+  @FXML
+  public ImageView avatarImageView;
+  @FXML
+  public ImageView countryImageView;
   @FXML
-  public JFXListView<MatchmakingQueue> queueListView;
+  public Label clanLabel;
+  @FXML
+  public Label usernameLabel;
+  @FXML
+  public Label teamRatingLabel;
+  @FXML
+  public Label gameCountLabel;
+  public Label ladderRatingLabel;
+  public HBox queueBox;
+  public PartyMemberItemController controller;
+  public FlowPane partyMemberPane;
+  private Player player;
 
   @Override
   public void initialize() {
 
-    playerListView.setCellFactory(listView -> new PartyMemberItemListCell(uiService));
-    playerListView.setItems(teamMatchmakingService.getParty().getMembers());
+    player = playerService.getCurrentPlayer().get();
+    countryImageView.imageProperty().bind(createObjectBinding(() -> StringUtils.isEmpty(player.getCountry()) ?
+        countryFlagService.loadCountryFlag("").orElse(null) // loads earth flag
+        : countryFlagService.loadCountryFlag(player.getCountry()).orElse(null), player.countryProperty()));
```

Comment:
```countryFlagService.loadCountryFlag(StringUtils.isEmpty() ? ... : ...)```

Reply:
Sorry, I don't get it

Reply:
This is very hard to read as you have duplicated code. You don't need to list countryFlagService.loadCountryFlag() twice. Just do the teneray operator on the country string.